sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -393,18 +394,19 @@ def list_databases(  # pylint:disable=docstring-missing-param
         *,
         session_token: Optional[str] = None,
         initial_headers: Optional[Dict[str, str]] = None,
+        response_hook: Optional[Callable[[Mapping[str, Any]], None]] = None,
         **kwargs: Any
     ) -> ItemPaged[Dict[str, Any]]:
         """List the databases in a Cosmos DB SQL database account.
 
         :param int max_item_count: Max number of items to be returned in the enumeration operation.
         :keyword str session_token: Token for use with Session consistency.
         :keyword Dict[str, str] initial_headers: Initial headers to be sent as part of the request.
-        :keyword Callable response_hook: A callable invoked with the response metadata.
+        :keyword response_hook: A callable invoked with the response metadata.
+        :paramtype response_hook: Callable[[Dict[str, str]], None]
```

Comment:
To match the type hint
```suggestion
        :paramtype response_hook: Callable[[Mapping[str, str]], None]
```

Reply:
Good catch! Fixed!